Sources/SkeletonView.swift

```diff
@@ -105,6 +158,13 @@ extension UIView {
         status = .on
     }
     
+    func updateSkeletonLayer(usingColors colors: [UIColor], gradientDirection direction: GradientDirection? = nil, animated: Bool, animation: SkeletonLayerAnimation? = nil) {
+        guard skeletonLayer != nil else { return }
```

Comment:
we could create a local let inside the guard, in order to remove the force unwrap we have in below lines. 
```
guard let layer = skeletonLayer else { return }
        layer.update(usingColors: colors)
        if animated { layer.start(animation) } else { layer.stopAnimation() }
        status = .on
```
What do you think?

Reply:
Done! Thanks for pointing it out.